Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.12.8-alpine
+LABEL mainteiner="sberdianskyi@mate.com"
```

Comment:
There is a typo in the LABEL instruction. It should be 'maintainer' instead of 'mainteiner'.